cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/CgmesExportUtil.java

```diff
@@ -213,6 +233,22 @@ public static String loadClassName(LoadDetail loadDetail) {
         return CgmesNames.ENERGY_CONSUMER;
     }
 
+    private static boolean isConformLoad(LoadDetail loadDetail) {
```

Comment:
Are you sure about this definition? The fixed powers can be non zero, no?

Reply:
The definition has not been changed from the previous one. The condition just has been moved to a specific method. If the load is not considered as "pure conform" (only variable part) or "pure non-conform" (only fixed part) it is declared as "non-conform", but this is not seen here now.

I agree that, if we decide to consider as "conform" also loads with variable part that may have a fixed part, it should be explicit in these conditions.

Then we should express it as: 

`isConform === loadDetail != null && (loadDetail.variableP != 0 || loadDetail.variableQ != 0)`

no matter which values of `fixedP`, `fixedQ`.

What do yo think @marqueslanauja ?

Reply:
As you said the definition has not been changed. If we decide to change it I think is better to do it in a separated PR.

Reply:
@nemanja-st what is the good definition of conform loads?

Reply:
@annetill `ConformLoad` is determined by the class name.
`EnergyConsumer.p` and `q` are the setpoints for all load instances so they cannot be used to differentiate if a load is fixed or variable.
Also, [CGM AC IG 1.3](https://eepublicdownloads.entsoe.eu/clean-documents/CIM_documents/Grid_Model_CIM/CGM_BUILDING_PROCESS_IMPLEMENTATION_GUIDE_AC_PART_1.3.pdf) **10.21** specifies that `EnergyConsumer.pfixed` and `qfixed` are not used in ENTSOe exchanges.

Reply:
Yes okay, so maybe we should throw or log and keep it like this.

Reply:
Agree.

And by the way, the IIDM `LoadDetail` fixed and variable parts are NOT related to CGMES `EnergyConsumer.pfixed,qfixed` values.

When importing from CGMES, if the energy consumer is a `ConformLoad`, its `p,q` setpoints are set in the `LoadDetail` variable part. If the energy consumer is a `NonConformLoad`, they are copied to the fixed part. See `EnergyConsumerConversion::setLoadDetail`.